src/GraphWebhooks/Startup.cs

```diff
@@ -81,21 +97,28 @@ public void ConfigureServices(IServiceCollection services)
             .AddSignalR(options => options.EnableDetailedErrors = true)
             .AddJsonProtocol();
 
-        services.AddMvc(options => {
+        services.AddMvc(options =>
+        {
             var policy = new AuthorizationPolicyBuilder()
                 .RequireAuthenticatedUser()
                 .Build();
             options.Filters.Add(new AuthorizeFilter(policy));
         })
+
         // Add the Microsoft Identity UI pages for signin/out
         .AddMicrosoftIdentityUI();
     }
 
-    // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+    /// <summary>
+    /// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+    /// </summary>
+    /// <param name="app">The application builder provided by the runtime.</param>
+    /// <param name="env">The web host environment provided by the runtime.</param>
+    /// <exception cref="ArgumentException">Thrown if any parameter is null.</exception>
     public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
     {
-        _ = app ?? throw new ArgumentException(nameof(app));
-        _ = env ?? throw new ArgumentException(nameof(env));
+        _ = app ?? throw new ArgumentException("IApplicationBuilder cannot be null", nameof(app));
+        _ = env ?? throw new ArgumentException("IWebHostEnvironment cannot be null", nameof(env));
 
```

Comment:
I'm not sure this change is helpful?
Instead of giving you the parameter name, it'll give you the type in the exception message

Reply:
It was complaining that I didn't use this form (https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2208).

I just tested it, and the message becomes `IApplicationBuilder cannot be null (Parameter 'app')`